h5py/h5t.pyx

```diff
@@ -1006,6 +1006,8 @@ cdef class TypeCompositeID(TypeID):
         """
         return H5Tget_member_index(self.id, name)
 
+def get_third(iterable):
```

Comment:
Dunno if there's a better place to put this. Unfortunately it's needed because cython 0.19 doesn't support lambdas inside cdef function definitions.